public/css/icinga/dashboards.less

```diff
@@ -0,0 +1,333 @@
+  color: @text-color-on-icinga-blue;
+  background-color: @icinga-blue;
+}
+
+.collapsed .action-link.add-dashlet, .collapsed .action-link.add-dashboard {
+  display: none;
+}
+
+.action-link.add-dashboard {
+  width: 85%;
+  margin: .5em 0 0;
+}
+
+.action-link.add-dashlet, .action-link.add-dashboard {
+  .rounded-corners();
+  order: 1;
+  font-size: 1.2em;
+  font-weight: 500;
+  padding: .6em 1em;
+  text-decoration: none;
+  background-color: @low-sat-blue;
+}
+
+.dashboard-list-control .dashlet-item-list {
+  margin-left: 2.2em;
+  display: flex;
+  width: 97.8%; // 100% - 2.2em margin-left
+
+  .dashlet-list-item {
+    width: 49.7%;
+    margin-bottom: .5em;
+    background: transparent;
+  }
+
+  .dashlet-list-item:nth-child(odd) {
+    margin-right: .5em;
+  }
+}
+
+.dashboard-list-control {
+  width: 85%;
+
+  h1.collapsible-header {
+    margin-left: 1.5em;
+  }
+
+  .action-link.add-dashlet {
+    width: 97.8%;
+    margin-left: 1.8em;
+  }
+}
+
+.home-list-control, .dashboard-list-control {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+
+  h1.collapsible-header {
+    border-bottom: none;
+  }
+
+  h1.collapsible-header a:not(.dashlet-title), h1.dashlet-header a:not(.dashlet-title) {
+    margin-left: .3em;
+    font-weight: normal;
+    color: @icinga-blue;
+  }
+
+  h1.dashlet-header a.dashlet-title {
+    color: @text-color;
+  }
+
+  .dashlets-list-info, .dashboard-list-info {
+    .expand-icon {
+      margin-top: .9em;
+    }
+
+    .collapse-icon {
+      margin-top: .6em;
+    }
+  }
+}
+
+// Dashboard home list controls only
+.home-list-control {
+  width: 100%;
+
+  & > h1.collapsible-header {
+    margin-left: 1.5em;
+    margin-bottom: -1em;
+  }
+
+  .dashboard-list-control, .action-link.add-dashboard {
+    margin-left: 1.4em;
+  }
+
+  .dashboard-list-control {
+    .dashboard-item-list, .dashlet-item-list {
+      margin: 0;
+    }
+
+    .dashlet-item-list {
+      margin-left: 2em;
+      width: 98%; // 100% - 2em margin-left
+    }
+
+    .action-link.add-dashlet {
+      margin-left: 1.6em;
+      width: calc((49.74% * 2) - 1.2em); // 2x flex columns - own size
+    }
+  }
+}
+
+// Add dashlets mini wizard
+.form-list-control {
+  span {
+    font-size: 1.3em;
+    opacity: .8;
+    font-weight: 500;
+  }
+}
+
+.form-list-control.multi-dashlets {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  border-top: 2px solid @gray-light;
+
+  .control-group {
+    width: 100%;
+  }
+}
+
+.form-list-control.multi-dashlets:not(.collapsed) span {
+  margin-left: 1.4em;
+}
+
+// Make the submit buttons a bit bigger than the normal size
+.modal-form.icinga-controls .control-group.form-controls input[type="submit"],
+.control-group.form-controls .modal-cancel {
+  min-width: 7em;
+  font-size: 1.2em;
+  padding: ~"calc(@{vertical-padding} - 2px)" @horizontal-padding;
+}
+
+// Remove and modal cancel buttons
+form.icinga-form .control-group.form-controls .remove-button {
```

Comment:
The class `remove-button` is to common to be used in such a equally common rule. Consider that anyone else might use this class at the same location. You cannot assume that the button then also requires an automatic right margin.

There's already `.btn-remove`, use that instead. You could also add the right margin to its definition in form.less. Why shouldn't remove buttons not always be aligned to the left? There take also a look into the reporting module's report modification form where such a button also exists and it wouldn't look bad (imho) if this applies there as well.

Reply:
Should the `.cancel-button()` mixin be applied to all `btn-remove` buttons by default instead of the `.button()` mixin. Because it makes no sense at all to have cancel buttons styled **red**, but not remove buttons.

Reply:
Forget `.cancel-button()`. It's mis-named, yes. Just apply `btn-remove` and it's fine.